testingOrganizations.json

```diff
@@ -0,0 +1,56 @@
+{
+  "organizations": [
+    {
+      "title": "Society of St. Vincent de Paul Archdiocese",
+      "address": "2403 Holcombe Blvd, Houston, TX 77021",
+      "phone": "(713) 741-8234",
+      "email": "info@svdphouston.org",
+      "description": "The Society of St. Vincent de Paul is a non-profit organization offering basic-needs assistance to the local community in need. Assistance is given through our Food Network, Clothing & Furniture Voucher, Disaster Response, and Rent & Utility Assistance Programs. The Society has been serving the local community since 1871.",
+      "categories": ["clothing", "vehicles", "furniture"],
+      "isSupportDropOff": true,
+      "isSupportMailIn": true,
+      "isSupportPickUp": true,
+      "latitude": 29.7052762,
+      "longitude": -95.3847942
```

Comment:
Are we also planning to support Google Maps' `placeId`? https://docs.google.com/document/d/1I_tWFGCNkeb4DWYDpN4AejZy5fS-55u7UGtlwl3iqAE/edit?disco=AAAAJts61F8

Reply:
Is there a way to find the placeIDs for these places online?

Reply:
You can use the *Place Id Finder* tool on the GCP's Place Id documentation page: https://developers.google.com/places/place-id

Reply:
Added 👍

Reply:
Thanks for updating it! Also looks like `latitude` and `longitude` fields are not included in the TDD. Should we dynamically get `latitude` and `longitude` by `placeId` at render-time? Since `latitude` and `longitude` can change and `placeId` is fixed per org. 

If we don't plan to implement `placeId` APIs in prototype, we can include a TODO here to remove `latitude` and `longitude` later.